apps/api/src/app/shared/services/cache/cache-service.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { expect } from 'chai';
+import { CachingConfig, ICacheService } from './cache.service';
+
+describe('cache-service', function () {
```

Comment:
this test should verify the cache service class logic not the interface

Reply:
At the moment I removed those tests due reason that here i wanted to test only the decorators, but at the moment the e2e usecasese tests are already testing them, would love to jump on a call and discuss in order to see if we need more test.